### ISSUE TITLE
Lycanthrope can not explore land

### ISSUE THREAD

**Issue by r00tmebaby** [WaveHack/OpenDominion#759]
In regards to Odarena (may differ from opendominion), I found two bugs so far.

Lycanthrope can not explore land. Receiving error 500. 
Simian diamond mines do not produce gems. 

## Context
I built 200 diamond mines on Simian under the protection and had to destroy them, rezone the land and rebuild it as the diamond mines did not produce any gems.

## Your Environment
* Odarena
* Chrome
* Windows

**Comment by Dr-Eki** [comment]
For the future, OD Arena issues are better opened in the ODA repo: https://github.com/Dr-Eki/OpenDominion

First bug, regarding exploration, has been fixed. 

For Simian, the bug was incorrect spell description (I forgot to update the text, but it's done now). The spell description should have said that there is no ore or gem production during Rainy Season. It's been fixed now. I've added 650,000 platinum to your Dominion to compensate for the cost of building and rebuilding.
